plasmapy/formulary/quantum.py

```diff
@@ -35,6 +35,17 @@ def deBroglie_wavelength(V: u.m / u.s, particle) -> u.m:
     r"""
     Calculates the de Broglie wavelength.
 
+    The de Broglie wavelength is given by
+
+    .. math::
+
+        \lambda_{dB} = \frac{h}{p} = \frac{h}{\gamma m V}
+
+    where :math:`h` is the Planck constant, :math:`p` is the
+    relativistic momentum of the particle, :math:`gamma` is the
+    Lorentz factor, :math:`m` is the particle's mass, and :math:`V` is the
+    particle's velocity.
```

Comment:
```suggestion
    Calculates the de Broglie wavelength.  The de Broglie wavelength is given by

    .. math::

        \lambda_{dB} = \frac{h}{p} = \frac{h}{\gamma m V}

    where :math:`h` is the Planck constant, :math:`p` is the
    relativistic momentum of the particle, :math:`\gamma` is the
    Lorentz factor, :math:`m` is the particle's mass, and :math:`V` is the
    particle's velocity.
```

Reply:
I think it is best to keep the one-line summary of the functions separate because it's a bit of a special / idiomatic thing that readers of docstrings might be expecting at the top. (It's also how readers see the functions being summarized in the [overview](https://plasmapy--951.org.readthedocs.build/en/951/formulary/quantum.html#functions) page.)

Reply:
Either way, but both options will display the same on the summary page.  The `.. autosummary::` and `.. audomodapi::` directives that generates those tables automatically takes only the first sentence of the docstring.